staging/src/k8s.io/legacy-cloud-providers/openstack/openstack_instances.go

```diff
@@ -157,37 +158,6 @@ func (i *Instances) InstanceShutdownByProviderID(ctx context.Context, providerID
 	return false, nil
 }
 
-// InstanceMetadataByProviderID returns metadata of the specified instance.
-func (i *Instances) InstanceMetadataByProviderID(ctx context.Context, providerID string) (*cloudprovider.InstanceMetadata, error) {
```

Comment:
Safe to remove since instance v2 is disabled:

https://github.com/kubernetes/kubernetes/blob/d3c1e81d5e6796db9389cd73073f48d6a37c4621/staging/src/k8s.io/legacy-cloud-providers/openstack/openstack_instances.go#L65-L68